pkg/asset/cluster/tfvars.go

```diff
@@ -286,6 +286,11 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 			ServiceAccount: string(sess.Credentials.JSON),
 		}
 
+		licenses := []string{}
+		for _, license := range installConfig.Config.GCP.Licenses {
+			licenses = append(licenses, license)
+		}
+
```

Comment:
why the copy? isn't it already a string slice?

Reply:
You're right.